src/nvim/lua/executor.c

```diff
@@ -1249,11 +1249,27 @@ bool nlua_exec_file(const char *path)
 {
   lua_State *const lstate = global_lstate;
 
-  if (luaL_loadfile(lstate, path)) {
+  lua_getglobal(lstate, "loadfile");
+  lua_pushstring(lstate, path);
+
+  lua_call(lstate, 1, 2);
```

Comment:
let's use `lua_pcall` to be sure. A user override might cause an uncaught exception. (we can use the same error format E5112 for this)

Reply:
Used the slightly more generic `E5111`.